src/encoded/tests/data/inserts/pipeline.json

```diff
@@ -166,17 +166,5 @@
         "award": "U54HG007005",
         "title": "Test STARR-seq pipeline",
         "uuid": "5fe3fdf6-5b1a-4aca-8116-ded7acf13037"
-    },
```

Comment:
I am not sure why this pipeline was removed form the inserts?

Reply:
I must've accidentally deleted it, will add it back